### ISSUE TITLE
csvgrep (others commands?) fail to quote output with vertical tab characters in input columns

### ISSUE THREAD

**Issue by johnnaegle** [wireservice/csvkit#483]
I'm operating on the NPPES monthly CSV download (~4.8 million records) and am filtering and trimming the data using csvgrep and csvcut.  I pipe the results together.  It seems that when a cell contains a vertical tab (character code 11), the piping causes the data to be modified:

For instance, with [small file](https://github.com/onyxfish/csvkit/files/97850/fred.csv.gz), I run this command to see what data is in the raw file:

```
csvstat -c "Last Update Date" fred.csv.gz 
 38. Last Update Date
    <type 'datetime.date'>
    Nulls: False
    Values: 2010-02-26, 2008-06-25, 2013-03-27

Row count: 3
```

so in fred.csv, Last Update Date has three values.  

If I add a csvgrep and pipe it to csvstat:

```
csvgrep -c "Entity Type Code" -m "1" fred.csv.gz  | csvstat -c "Last Update Date"
 38. Last Update Date
    <type 'unicode'>
    Nulls: False
    Values: Y, 02/26/2010

Row count: 2
```

After the grep, I have a value (Y) in that column that does not appear in the original data.  It seems that CSV grep is not detecting that it needs to output a cell with a vertical tab with escaping.   There doesn't seem to be a way to force each cell to use quoting in the output.

Is there a way to force quoting in the output?

**Comment by jpmckinney** [comment]
I can reproduce this.

**Comment by jpmckinney** [comment]
Adding a reminder to re-test this once #496 is closed

**Comment by onyxfish** [comment]
Fixed in csvkit `1.0.0`
